### ISSUE TITLE
Include option to add file to existing ZIP

### ISSUE THREAD

**Issue by Ecksters** [nfriedly/node-bestzip#60]
Would love an option to add new files to an existing ZIP file.

**Comment by nfriedly** [comment]
Yeah, that's not a bad idea. Is the behavior consistent across various zip implementations? If so, would you be willing to code it up along with some tests?

**Comment by Ecksters** [comment]
One issue I see is that Archiver doesn't appear to support it, so we'd potentially need to change the library used for fallback.
